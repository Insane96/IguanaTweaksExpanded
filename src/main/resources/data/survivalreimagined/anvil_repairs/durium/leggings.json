{
	"item_to_repair": {
		"id": "survivalreimagined:durium_leggings"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 7,
			"max_repair": 0.5,
			"cost_multiplier": 0.5
		},
		{
			"repair_material": {
				"id": "survivalreimagined:durium_ingot"
			},
			"amount": 2
		},
		{
			"repair_material": {
				"id": "survivalreimagined:durium_nugget"
			},
			"amount": 18
		}
	]
}