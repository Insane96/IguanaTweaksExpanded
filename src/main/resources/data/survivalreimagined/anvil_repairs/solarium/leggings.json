{
	"item_to_repair": {
		"id": "survivalreimagined:solarium_leggings"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 7,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "survivalreimagined:solarium_ball"
			},
			"amount": 2
		}
	]
}