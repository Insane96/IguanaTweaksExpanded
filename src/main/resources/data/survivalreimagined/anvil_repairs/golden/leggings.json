{
	"item_to_repair": {
		"id": "minecraft:golden_leggings"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:golden_ingot"
			},
			"amount": 7
		},
		{
			"repair_material": {
				"id": "minecraft:golden_nugget"
			},
			"amount": 63
		}
	]
}